### ISSUE TITLE
Failed when activated for the second time in the keep-alive component

### ISSUE THREAD

**Issue by zsedcbm** [David-Desmaisons/Vue.resize#10]
When I wrap the div with the keep-alive component, v-resize is valid the first time it is entered. When the div re-triggers activated, v-resize is invalid.

**Comment by zsedcbm** [comment]
[https://codepen.io/zsedcbm/pen/KJaNRO](https://codepen.io/zsedcbm/pen/KJaNRO)
![resize](https://user-images.githubusercontent.com/5986396/52033931-b3328a00-2561-11e9-8a58-8b9fc7054348.gif)

**Comment by David-Desmaisons** [comment]
I found this workaround:
```
  activated(){
    Vueresize.inserted(this.$el, { value: this.relayout});
  },
  deactivated(){
    Vueresize.unbind(this.$el, { value: this.relayout});
  },
```
Not the most elegant solution, but for now no better idea 

https://codepen.io/dede89/pen/JzvPZZ

**Comment by David-Desmaisons** [comment]
Corrected in version 1.2.0
